resources/common-toc.ditamap

```diff
@@ -42,6 +42,7 @@
   </topicref>
   <topicref keyref="dita-and-dita-ot-resources">
     <mapref href="../topics/dita-resources.ditamap" toc="no"/>
+    <mapref href="../topics/dita-ot-day-videos.ditamap"/>
```

Comment:
@infotexture, thoughts on adding `@toc="no"` here?

Reply:
I'd be inclined to keep this as it is, since users may be interested in browsing by year, and the ToC entries make this easier and more discoverable.